packages/conjury/conjury.2.1.1/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+synopsis: "Conjury library for OMake"
+maintainer: "james woodyatt <jhw@conjury.org>"
+authors: "james woodyatt <jhw@conjury.org>"
+license: "BSD-2-Clause"
+tags: "org:conjury.org"
+homepage: "https://bitbucket.org/jhw/conjury/"
+bug-reports: "https://bitbucket.org/jhw/conjury/issues"
+depends: [
+  "base-unix"
+  "ocaml" { >= "4.08"}
+  "ocamlfind" { >= "1.7.3"}
+  "omake" { = "0.10.3"}
```

Comment:
```suggestion
  "omake" { = "0.10.5"}
```

Reply:
Ah. I think I see my mistake. I inadvertently changed the `>=` to `=` and I need to change it back. I think OMake 0.10.5 is still broken on OCaml 5.0 anyway.